### PR TITLE
rename all bucket commands

### PR DESCRIPTION
**Pull request by harshavardhana**

### CODE REVIEW

File: cmd/versioning-suspend.go [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /*
  * MinIO Client (C) 2020 MinIO, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Versioning 2.0 (the "License");
```
Comment (kannappanr) [inline cmd/versioning-suspend.go:4]:
```suggestion
 * Licensed under the Apache License, version 2.0 (the "License");
```

File: cmd/versioning-main.go [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /*
  * MinIO Client (C) 2020 MinIO, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Versioning 2.0 (the "License");
```
Comment (kannappanr) [inline cmd/versioning-main.go:4]:
```suggestion
 * Licensed under the Apache License, version 2.0 (the "License");
```

File: cmd/versioning-info.go [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /*
  * MinIO Client (C) 2020 MinIO, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Versioning 2.0 (the "License");
```
Comment (kannappanr) [inline cmd/versioning-info.go:4]:
```suggestion
 * Licensed under the Apache License, version 2.0 (the "License");
```

File: cmd/versioning-enable.go [hunk 4]
```diff
@@ -1,7 +1,7 @@
 /*
  * MinIO Client (C) 2020 MinIO, Inc.
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
+ * Licensed under the Apache License, Versioning 2.0 (the "License");
```
Comment (kannappanr) [inline cmd/versioning-enable.go:4]:
```suggestion
 * Licensed under the Apache License, version 2.0 (the "License");
```

File: cmd/versioning-info.go [hunk 5]
```diff
@@ -28,69 +28,69 @@ import (
...
 USAGE:
   {{.HelpName}} TARGET
- 
+
 FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 EXAMPLES:
-   1. Display bucket versioning status for bucket "mybucket".
+   1. Display bucket versioninging status for bucket "mybucket".
```
Comment (poornas) [inline cmd/versioning-info.go:47]:
```suggestion   
   1. Display bucket versioning status for bucket "mybucket".
```

File: cmd/versioning-info.go [hunk 6]
```diff
@@ -28,69 +28,69 @@ import (
...
-type versionInfoMessage struct {
+type versioningInfoMessage struct {
 	Op         string
 	Status     string `json:"status"`
 	URL        string `json:"url"`
 	Versioning struct {
 		Status    string `json:"status"`
 		MFADelete string `json:"MFADelete"`
-	} `json:"versioning"`
+	} `json:"versioninging"`
```
Comment (poornas) [inline cmd/versioning-info.go:66]: s/versioninging/versioning/g

**Pull request merged**